.github/workflows/main.yml

```diff
@@ -19,8 +19,8 @@ jobs:
       - name: Get Date
         id: get-date
         run: |
-          echo "::set-output name=date::$(/bin/date -u "+%Y%m%d")"
-          echo "::set-output name=yyyymm::$(/bin/date -u "+%Y%m")"
+          echo "date=$(/bin/date -u "+%Y%m%d")" >> $GITHUB_OUTPUT
+          echo "yyyymm=$(/bin/date -u "+%Y%m")" >> $GITHUB_OUTPUT 
```

Comment:
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/